templates/question_rate.mustache

```diff
@@ -135,11 +135,11 @@
     {{#qelements.rows}}
     <tr class="raterow">
       {{#cols}}
-      <td {{#colstyle}}style="{{.}}"{{/colstyle}}{{#colclass}} class="{{.}}"{{/colclass}}{{#coltitle}} title="{{.}}"{{/coltitle}} 
+      <td {{#colstyle}}style="{{.}}"{{/colstyle}}{{#colclass}} class="{{.}}"{{/colclass}}{{#coltitle}} title="{{.}}"{{/coltitle}}
 			{{#colinput}} onclick="if('{{id}}'.length>0) document.getElementById('{{id}}').checked = true"{{/colinput}}>
         {{#coltext}}{{{.}}}{{/coltext}}
         {{#colhiddentext}}<span class="accesshide">{{{.}}}</span>{{/colhiddentext}}
-        {{#colinput}}<input type="radio" name="{{name}}" id="{{id}}" value="{{value}}"{{#checked}} checked="checked"{{/checked}}{{#disabled}} disabled="disabled"{{/disabled}}{{#onclick}} onclick="{{.}}"{{/onclick}}>
+        {{#colinput}}<label><input type="radio" name="{{name}}" id="{{id}}" value="{{value}}"{{#checked}} aria-label="{{#coltext}}{{{.}}}{{/coltext}}" checked="checked"{{/checked}}{{#disabled}} disabled="disabled"{{/disabled}}{{#onclick}} onclick="{{.}}"{{/onclick}}></label>
```

Comment:
Are you sure that this is correct? Is there any reason for the explicit label as well as the aria-label? Especially since the explicit label doesn't have any descriptive text

Reply:
ran this in axe dev tools, the issue was to wrap the radio buttons inside the <label> the errors were removed on the axe dev tools when wrapped like this

Reply:
Is colinput an iterable? So like are there multiple radio buttons?

Reply:
yes, so all those inside the colinput are wrapped between label

Reply:
Would suggest creating a separate label element (not wrapping) and then associating it with the for label attribute. Can also remove the aria-label attribute as it will have a label programatically associated with it